compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/codegen/ExpressionCodegen.kt

```diff
@@ -919,6 +975,9 @@ class ExpressionCodegen(
         }
 
         if (tryCatchBlockEnd != null) {
+            if (tryInfo != null) {
+                tryInfo.tryBlock.finallyExpression!!.markLineNumber(startOffset = false)
```

Comment:
This line is not clear, do we really need it?

Reply:
Added this to match genFinallyBlockOrGoto(), (see old ExpressionCodegen.java#1537.). I am not sure this is actually hit by any tests. Feel free to remove.